src/goGetPackage.ts

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+import vscode = require('vscode');
+import cp = require('child_process');
+import path = require('path');
+import fs = require('fs');
+import { getGoRuntimePath } from './goPath';
+import { getCurrentGoPath, getImportPath } from './util';
+
+export function goGetPackage() {
+	const editor = vscode.window.activeTextEditor;
+	const selection = editor.selection;
+	const selectedText = editor.document.lineAt(selection.active.line).text;
+
+	const importPath = getImportPath(selectedText);
+	if (importPath === '') {
+		vscode.window.showErrorMessage('No import path to get');
+		return;
+	}
+
+	const goRuntimePath = getGoRuntimePath();
+	if (!goRuntimePath) {
+		return vscode.window.showErrorMessage('Could not locate Go binaries. Make sure you have Go installed');
+	}
+
+	fs.exists(path.join(getCurrentGoPath(), 'src', importPath), (exists) => {
```

Comment:
getCurrentGoPath() can return `:` or `;` separated paths as well. This would fail then

Reply:
Instead you can run `go list` and pass the importpath to see if the package exists.
Or skip this check altogether.

I really feel that if a user has explicitly run this `Go: Get Package` command then they would know that this would get the package using `go get`